pkg/cmd/roachtest/tests/mixed_version_backup.go

```diff
@@ -2602,6 +2610,46 @@ func bankWorkloadCmd(
 	return init, run
 }
 
+func schemaChangeWorkloadCmd(
+	l *logger.Logger, testRNG *rand.Rand, roachNodes option.NodeListOption, mock bool,
+) (init *roachtestutil.Command, run *roachtestutil.Command) {
+	maxOps := 1000
+	concurrency := 5
+	if mock {
+		maxOps = 10
+		concurrency = 2
+	}
+	initCmd := roachtestutil.NewCommand("./workload init schemachange").
+		Arg("{pgurl%s}", roachNodes)
+	runCmd := roachtestutil.NewCommand(fmt.Sprintf("COCKROACH_RANDOM_SEED=%d ./workload run schemachange", testRNG.Int63())).
+		Flag("verbose", 1).
+		Flag("max-ops", maxOps).
+		Flag("concurrency", concurrency).
+		Arg("{pgurl%s}", roachNodes)
+	l.Printf("sc init: %s", initCmd)
+	l.Printf("sc run: %s", runCmd)
+	return initCmd, runCmd
+}
+
+func prepSchemaChangeWorkload(
```

Comment:
Maybe this is obvious to DR, but it's not clear to me how these statements are "preparing" the workload.